integreat_cms/static/src/js/ajax-contact-form.ts

```diff
@@ -31,11 +32,24 @@ const showMessage = (data: any) => {
         setTimeout(() => {
             successMessageField.classList.add("hidden");
         }, timeoutDuration);
-    } else {
+    } else if (data.contact_form.length > 0) {
         const errorMessageField = document.getElementById("contact-ajax-error-message");
-        errorMessageField.classList.remove("hidden");
+        data.contact_form.forEach((error: any) => {
+            const node = document.createElement("div");
+            node.classList.add("bg-red-100", "border-l-4", "border-red-500", "text-red-700", "px-4", "py-3");
```

Comment:
With multiple error messages able to pop up simultaneously, we should put some minimal space between them so they don't appear as one visual object

```suggestion
            node.classList.add("bg-red-100", "border-l-4", "border-red-500", "text-red-700", "px-4", "py-3", "my-1");
```